doc/miscellaneous.md

```diff
@@ -0,0 +1,40 @@
+# Miscellaneous
+
+## Dataset filter
+
+In order to filter datasets in scenario creation dialog (for scenario master only), there are two options:
+
+- API (Recommended)
+- Dataset filter (Deprecated)
+
+### API (Recommended)
+
+All dataset that are needed to be selectable in dataset list must be linked to the matching workspace.
+In the same way, datasets that are needed not to be visible must be unlinked from the workspace.
+
+This consists of using **_workspace/dataset_** `/link` - `/unlink` API endpoint
+
+### Dataset Filter workspace option (Deprecated)
+
+Datasets included in the `[workspace].webApp.options.datasetFilter` array attribute will be added in the datasets list in addition with linked datasets. So it is recommended to let this attribute as _null_
```

Comment:
in addition **to** :slightly_smiling_face: 
Also, I think, it is better to rephrase : So it is recommended to **set** this attribute **to** _null_ or So it is recommended to **leave** this attribute _null_